bundles/best_practices.rst

```diff
@@ -546,6 +546,120 @@ Beware that templates use a simplified version of the logical path shown above.
 For example, an ``index.html.twig`` template located in the ``templates/Default/``
 directory of the AcmeBlogBundle, is referenced as ``@AcmeBlog/Default/index.html.twig``.
 
+Specifics for a UX Bundle
+-------------------------
+
+Here are a few tricks to make your bundle install as a UX bundle.
+
+composer.json file
+~~~~~~~~~~~~~~~~~~
+
+Your ``composer.json`` file must have the ``symfony-ux`` keyword:
+
+.. code-block:: json
+
+    {
+        "keywords": ["symfony-ux"]
+    }
+
+Assets location
+~~~~~~~~~~~~~~~
+
+Your assets must be located in one of the following directories, with a ``package.json`` file so Flex can handle it
+during install/update:
+
+* ``/assets`` (recommended)
+* ``/Resources/assets``
+* ``/src/Resources/assets``
+
+package.json file
+~~~~~~~~~~~~~~~~~
+
+Your ``package.json`` file must contain a ``symfony`` config with controllers defined, and also add required packages
+to the ``peerDependencies``:
+
+.. code-block:: json
+
+    {
+        "name": "@acme/feature",
+        "symfony": {
+            "controllers": {
+                "slug": {
+                    "main": "dist/controller.js",
+                    "webpackMode": "eager",
+                    "fetch": "eager",
+                    "enabled": true,
+                    "autoimport": {
+                        "dist/bootstrap4-theme.css": false,
+                        "dist/bootstrap5-theme.css": true
+                    }
+                }
+            }
+        },
+        "peerDependencies": {
+            "@hotwired/stimulus": "^3.0.0",
+            "slugify": "^1.6.5"
+        }
+    }
+
+In this case, the file located at ``[assets directory]/dist/controller.js`` will be exposed.
+
+.. tip::
```

Comment:
I don't think such tip belongs to the page about bundle best practices. We might need a separate doc page.

Thus, those "specifics" are not even best practices. they are requirements for hooking into the Flex-managed stimulus controllers.

Reply:
Would you then recommend adding all these ux-specifics in a new, separate section of the docs?

Reply:
Moving to a new doc somewhere in the frontend section seems good to me. Then we can go into plenty of detail there.

Reply:
These new docs have been moved to a separate docs file, in frontend/ directory. :)